examples/061_TopLevelCantBeImplicitID.scala

```diff
@@ -1,6 +1,9 @@
-// I can't seem to reproduce this one. I see in `Checking.scala` that is should
-// be it if sym.owner.is(Package), but when playing around that doesn't seem to
-// ever be true
-// INCOMPLETE
+// LEGACY In the past top level implicit were not allowed. This is however no longer the case as of https://github.com/lampepfl/dotty/pull/5754
+// START
 @main def TopLevelCantBeImplicitID = ()
 
+import scala.language.implicitConversions
+case class C(str: String)
+implicit def toC(x: String): C = C(x)
+implicit val defaultC: C = C("default")
```

Comment:
I think we might be even able to just put the most simple example possible to prove this. I'm assuming that just a top level
```scala
implicit val foo: String = "foo"
```
should also provide the same example right? If so, let's just keep it as simple as possible.